superset-frontend/spec/javascripts/components/ListView/ListView_spec.jsx

```diff
@@ -347,10 +348,6 @@ describe('ListView with new UI filters', () => {
     expect(wrapper.find(ListViewFilters)).toHaveLength(1);
   });
 
-  it('fetched selects if function is provided', () => {
```

Comment:
fetching now happens on select open/click. This is really hard to simulate via enzyme.